tensorflow/core/util/util.cc

```diff
@@ -22,7 +22,6 @@ limitations under the License.
 #include "tensorflow/core/framework/device_factory.h"
 #include "tensorflow/core/lib/gtl/inlined_vector.h"
 #include "tensorflow/core/lib/strings/strcat.h"
-#include "tensorflow/core/platform/cpu_info.h"
```

Comment:
Please also remove the implementation of `IsMKLEnabled()` in this file.

Reply:
Thanks for your comment, removed it as per request